apps/src/templates/progress/ProgressLessonContent.story.jsx

```diff
@@ -11,10 +11,7 @@ export default storybook => {
         name:'progress lesson content',
         story: () => (
           <ProgressLessonContent
-            description={"At some point we reach a physical limit of how fast " +
-              "we can send bits and if we want to send a large amount of " +
-              "information faster, we have to finds ways to represent the same " +
-              "information with fewer bits - we must compress the data."}
```

Comment:
We don't actually use description anywhere right now, so it seems silly to have a storybook for it.